DTS-STN/digital-dojo-engagement
# Tools and Resources

## ADO-610 Create a Tools and Resources Page

### Description
This page is used to showcase to clients embedded YouTube videos and terminology related to various DevOps concepts.

List of proposed changes:

- add page
- add tests
- add translations

### What to test for/How to test
- open page
- make sure YouTube videos render and can be played
- click on the different letter buttons to switch between definitions
- click on definitions to have them toggle expanded or collapsed

### Additional Notes
